src/middlewared/middlewared/plugins/account.py

```diff
@@ -277,15 +276,13 @@ async def query(self, filters, options):
         if dssearch:
             ds_state = await self.middleware.call('directoryservices.get_state')
             if ds_state['activedirectory'] == 'HEALTHY' or ds_state['ldap'] == 'HEALTHY':
-                dssearch_results = await self.middleware.call('dscache.query', 'USERS', filters, options.copy())
+                ds_users = await self.middleware.call('dscache.query', 'USERS', filters, options.copy())
```

Comment:
Just to be sure, the users we get here will have all attrs our regular local db users have (normalized and everything) ?

Reply:
Yes.